WORKSPACE

```diff
@@ -0,0 +1,30 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
It would be nice if we could introduce rules_python via Bzlmod, but I guess there is still work to be done to adopt Bzlmod for rules_python?

Reply:
Definitely happy to collaborate on that, will send out the migration guide doc ASAP. I know you have a regular rules author SIG sync, feel free to add me to the meeting.

Reply:
Finally done :)